bin/redshift/setup_aws_redshift.sh

```diff
@@ -0,0 +1,72 @@
+
+REDSHIFT_CREATE_CLUSTER_OUTPUT=$(aws redshift create-cluster \
+  --db-name $REDSHIFT_DATABASE_NAME \
+  --region $REDSHIFT_AWS_REGION \
+  --availability-zone $REDSHIFT_AWS_AVAILABILITY_ZONE \
+  --node-type $REDSHIFT_NODE_TYPE \
+  --number-of-nodes 1 \
+  --master-username $REDSHIFT_USERNAME \
+  --master-user-password $REDSHIFT_MASTER_USER_PASSWORD \
+  --cluster-identifier $REDSHIFT_CLUSTER_IDENTIFIER \
+  --cluster-subnet-group-name $REDSHIFT_SUBNET_GROUP_NAME \
+  --cluster-type single-node\
+  --vpc-security-group-ids "$REDSHIFT_VPC_SECURITY_GROUP_IDS" \
+  --iam-roles $REDSHIFT_IAM_ROLES \
+  --automated-snapshot-retention-period 0 \
+  $REDSHIFT_CLUSTER_ACCESSIBILITY \
+  --tags Key=cloud,Value=aws Key=environment,Value=test Key=project,Value=trino-redshift Key=ttl,Value=$REDSHIFT_CLUSTER_TTL)
+
+echo "Creating the Amazon Redshift cluster $REDSHIFT_CLUSTER_IDENTIFIER on the region $REDSHIFT_AWS_REGION."
+
+# Wait for the cluster to become available
+aws redshift wait cluster-available \
+  --cluster-identifier $REDSHIFT_CLUSTER_IDENTIFIER
+
+echo "The Amazon Redshift cluster $REDSHIFT_CLUSTER_IDENTIFIER on the region $REDSHIFT_AWS_REGION is available for queries."
+
+echo $REDSHIFT_CLUSTER_IDENTIFIER > $DIR/.cluster-identifier
+
+set +xu
+REDSHIFT_CLUSTER_DESCRIPTION=$(aws redshift describe-clusters --cluster-identifier $REDSHIFT_CLUSTER_IDENTIFIER)
+REDSHIFT_CLUSTER_ENDPOINT=$(echo $REDSHIFT_CLUSTER_DESCRIPTION | jq -r '.Clusters[0].Endpoint.Address' )
+REDSHIFT_CLUSTER_PORT=$(echo $REDSHIFT_CLUSTER_DESCRIPTION | jq -r '.Clusters[0].Endpoint.Port' )
+REDSHIFT_CLUSTER_DATABASE_NAME=$(echo $REDSHIFT_CLUSTER_DESCRIPTION | jq -r '.Clusters[0].DBName' )
+REDSHIFT_CLUSTER_MASTER_USERNAME=$(echo $REDSHIFT_CLUSTER_DESCRIPTION | jq -r '.Clusters[0].MasterUsername' )
+
+
+# If running on GHA, export variables for future steps in GHA
+if [[ "${CONTINUOUS_INTEGRATION:-}" = "true" ]]; then
+    (echo "REDSHIFT_USER=$REDSHIFT_CLUSTER_MASTER_USERNAME"
+    echo "REDSHIFT_PASSWORD=$REDSHIFT_MASTER_USER_PASSWORD"
+    echo "REDSHIFT_ENDPOINT=$REDSHIFT_CLUSTER_ENDPOINT:$REDSHIFT_CLUSTER_PORT") >> "$GITHUB_ENV"
+else
+    echo "
+Steps to run tests interactively:
+
+psql command line:
+export PGPASSWORD='${REDSHIFT_MASTER_USER_PASSWORD}'
+psql -h ${REDSHIFT_CLUSTER_ENDPOINT%:*} -p 5439 -U ${REDSHIFT_CLUSTER_MASTER_USERNAME} ${REDSHIFT_CLUSTER_DATABASE_NAME}
+
+IntelliJ VM Arguments for Run Configuration:
+get those from maven test arguments below prefixed -Dtest.redshift., e.g. -Dtest.redshift.some.argument=\"1234\" and don't forget to remove trailing '\' after each argument
+
+Maven Test:
+./mvnw test \\
+    -B -Dair.check.skip-all=true -Dmaven.javadoc.skip=true \\
+    -Dtest.redshift.jdbc.user=\"${REDSHIFT_CLUSTER_MASTER_USERNAME}\" \\
+    -Dtest.redshift.jdbc.password=\"${REDSHIFT_MASTER_USER_PASSWORD}\" \\
```

Comment:
instead of printing password to the console, you can store it in a file and output `...=$(cat /tmp/.redshift-password-file)"` here